open-apparel-registry/open-apparel-registry
# Add a processing documentation for contributors

## Overview

Describes how CSVs are processed and facilities are matched in non-technical language.

Connects #268

### Demo

<img width="836" alt="Screen Shot 2019-03-21 at 10 13 30 AM" src="https://user-images.githubusercontent.com/17363/54771203-05874300-4bc2-11e9-9274-dc5bde0ea058.png">


<img width="1127" alt="Screen Shot 2019-03-21 at 10 12 24 AM" src="https://user-images.githubusercontent.com/17363/54771164-e983a180-4bc1-11e9-81af-927c1132eaa1.png">

<img width="1128" alt="Screen Shot 2019-03-21 at 10 12 31 AM" src="https://user-images.githubusercontent.com/17363/54771167-ebe5fb80-4bc1-11e9-9927-f86b3e1fd582.png">

<img width="1126" alt="Screen Shot 2019-03-21 at 10 12 52 AM" src="https://user-images.githubusercontent.com/17363/54771171-eee0ec00-4bc1-11e9-9133-a1fcda0e0e15.png">

## Testing Instructions

* Verify that there is an About > Processing option in the navbar and that it links to /about/processing
* Verify the content of /about/processing
* Log in
* Verify that /contribute includes a link to /about/processing
* Verify that the list details page includes a link to /about/processing
